lib/asciidoctor/substitutors.rb

```diff
@@ -971,11 +971,14 @@ def highlight_source source, process_callouts
   #
   # source - The String source.
   # spec   - The lines specifier (e.g., "1-5, !2, 10" or "1..5;!2;10")
+  # start  - The start attribute
   #
   # Returns an [Array] of unique, sorted line numbers.
-  def resolve_lines_to_highlight source, spec
+  def resolve_lines_to_highlight source, spec, start
```

Comment:
This method is used by Asciidoctor PDF, so this argument needs to be optional.

Reply:
`start = 1`

Reply:
> This method is used by Asciidoctor PDF, so this argument needs to be optional.

The value is not optional by default? In other words, the value will be `nil` if not defined no?

> start = 1

Ok :+1:

Reply:
Without the default value, it is required (unlike JavaScript) and thus will throw a runtime error.